Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -1523,6 +1738,146 @@ def get_context_table_csv(client: Client, args: Dict[str, str]) -> Tuple[Any, Di
+
+    users_response = session_details_raw_data.get('users', {})
+    for user_name, user_details in users_response.items():
+        user_info = user_details.get('info', {})
+        users = contents_append_notable_session_user_details(users, user_details, user_info)
+
+    executive_user = session_details_raw_data.get('executiveUserFlags', {})
+    for username, status in executive_user.items():
+        executive_user_flags.append({
+            username: status
+        })
+
+    contents_entry = {'sessions': contents, 'users': users, 'executiveUserFlags': executive_user_flags}
+
+    entry_context = {'Exabeam.NotableSession(val.SessionID && val.SessionID === obj.SessionID)': contents_entry}
+
+    human_readable = tableToMarkdown('Notable Session details:', contents_entry, removeNull=True)
+
+    return human_readable, entry_context, session_details_raw_data
+
+
+def get_notable_sequence_details(client: Client, args: Dict[str, str]) -> Tuple[Any, Dict[str, Any], Optional[Any]]:
+    """  Updates records of a context table.
+
+    Args:
+        client: Client
+        args: Dict
+
+    """
+    asset_id = args.get('asset_id')
+    start_time = args.get('start_time', '30 days ago')
+    end_time = args.get('end_time', '0 minutes ago')
+    parse_start_time = convert_date_to_unix(start_time)
+    parse_end_time = convert_date_to_unix(end_time)
+
+    sequence_details_raw_data = client.get_notable_sequence_details_request(asset_id, parse_start_time, parse_end_time)
+    if not sequence_details_raw_data:
+        return f'The Asset {asset_id} has no sequence details in this time frame.', {}, {}
+
+    contents: list = []
+    for sequence in sequence_details_raw_data:
+        sequence_info = sequence.get('sequenceInfo')
+        contents = contents_append_notable_sequence_details(sequence, sequence_info, contents)
+
+    entry_context = {'Exabeam.Sequence(val.sequenceId && val.sequenceId === obj.sequenceId)': contents}
+
+    human_readable = tableToMarkdown('Notable sequence details:', contents, removeNull=True)
+
+    return human_readable, entry_context, sequence_details_raw_data
+
+
+def get_notable_sequence_event_types(client: Client, args: Dict[str, str]) -> Tuple[Any, Dict[str, Any], Optional[Any]]:
+    """  Updates records of a context table.
+
+    Args:
+        client: Client
+        args: Dict
+
+    """
+    asset_sequence_id = args.get('asset_sequence_id')
+    search_str = args.get('search_str')
+    sequence_event_types_raw_data = client.get_notable_sequence_event_types_request(asset_sequence_id, search_str)
+
+    if not sequence_event_types_raw_data:
+        return f'The Asset {asset_sequence_id} has no sequence event types.', {}, {}
+
+    contents: list = []
+    for sequence in sequence_event_types_raw_data:
+        contents = contents_append_notable_sequence_event_types(sequence, contents)
+
+    entry_context = {'Exabeam.SequenceEventTypes(val.eventType && val.eventType === obj.eventType)': contents}
```

Comment:
I believe you need to add the sequence ID to the outputs and then filter by the sequence ID and not event type.
```suggestion
    entry_context = {'Exabeam.SequenceEventTypes(val.sequenceId && val.sequenceId === obj.sequenceId)': contents}
```

Reply:
Done